src/iris/api.py

```diff
@@ -3,7 +3,10 @@
 
 from __future__ import absolute_import
 
-from gevent import socket
+from gevent import monkey
+monkey.patch_all()  # NOQA
```

Comment:
Is this needed? I thought when you use uwsgi or gunicorn with gevent it automatically monkey patches everything.